gems/sorbet-runtime/lib/types/private/sealed.rb

```diff
@@ -5,8 +5,8 @@ module T::Private::Sealed
   module NoInherit
     def inherited(child)
       super
-      this_line = Kernel.caller.find {|line| !line.match(/in `inherited'$/)}
-      T::Private::Sealed.validate_inheritance(this_line, self, child, 'inherited')
+      this_line = Kernel.caller_locations.find {|loc| loc.base_label != 'inherited'}
```

Comment:
Would be nice to use `Thread.each_caller_location` here too, but in the grand scheme of things the performance shouldn't be huge because the stack generally isn't that big during boot time.